speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -162,12 +162,27 @@ void PControlTestMediator(bool p_intervention, bool p_gas, bool p_steer, bool p_
     ASSERT_TRUE(SetupSingletonsFolder());
     tParticipantControl arr = {p_intervention, p_gas, p_steer, p_force};
     SDAConfigMediator::GetInstance()->SetPControlSettings(arr);
-    const SDAConfig config = SDAConfigMediator::GetInstance()->GetDecisionMaker()->Config;
-    tParticipantControl pControl = config.GetPControlSettings();
+    tParticipantControl pControl = SDAConfigMediator::GetInstance()->GetPControlSettings();
     ASSERT_EQ(arr.ControlInterventionToggle, pControl.ControlInterventionToggle);
     ASSERT_EQ(arr.ControlSteering, pControl.ControlSteering);
     ASSERT_EQ(arr.ControlGas, pControl.ControlGas);
 
+/// @brief Tests the Mediator ParticipantControlSettings for every possible boolean combination (first 3)
```

Comment:
Lots of errors in this file.
- A missing '}' here
- Outdated tests below (main has the correct ones)